index.js

```diff
@@ -1,11 +1,21 @@
 'use strict';
 const fs = require('fs');
 const pify = require('pify');
+
 const fsP = pify(fs);
 const fsReadP = pify(fs.read, {multiArgs: true});
 
+function createBuffer(len) {
+	if (Buffer.alloc) {
+		return Buffer.alloc(len);
+	}
+
+	// TODO: remove when when Node.js 6 is target (needed for Node.js < 5.10.0)
```

Comment:
This was backported and I believe it should work for Node.js > 4.5 or something.